CvGameCoreDLL_Expansion2/CvPlayer.cpp

```diff
@@ -14244,120 +14318,123 @@ void CvPlayer::doGoody(CvPlot* pPlot, CvUnit* pUnit)
 
 	CvAssertMsg(pPlot->isGoody(), "pPlot->isGoody is expected to be true");
 
-	if(!isBarbarian())
-	{
-		m_bEverPoppedGoody = true;
-		pPlot->removeGoody();
+	// Barbarians don't claim ruins
+	if (isBarbarian())
+		return;
 
-		// Minors don't get Goodies :(
-		if(isMinorCiv())
-		{
-			return;
-		}
+	// Minors don't claim ruins in VP
```

Comment:
Is this a bugfix? Did Minors claim ancient ruins in the base game?

Reply:
They did in the old code (just can't get any reward from it).